src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -161,7 +161,6 @@ def validation_error(self, message):
     def error(self, message):
         telemetry.set_user_fault('parse error: {}'.format(message))
         self._handle_command_package_error(message)
-
         args = {'prog': self.prog, 'message': message}
         logger.error('%(prog)s: error: %(message)s', args)
         self.print_usage(sys.stderr)
```

Comment:
Changes in this file shouldn't be required for this feature.
Are they for progress?